python/tidy/servo_tidy/tidy.py

```diff
@@ -991,25 +1001,27 @@ def check_commits(path='.'):
     raise StopIteration
 
 
-def scan(only_changed_files=False, progress=True):
+def scan(only_changed_files=False, progress=True, stylo=False):
```

Comment:
We should override `only_changed_files` to `False` if we have `stylo`. That way, some of the functions (`FileList` and `filter_files`) don't need another argument.